awx/sso/strategies/django_strategy.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2017 Ansible, Inc.
+# All Rights Reserved.
+
+from social.strategies.django_strategy import DjangoStrategy
+
+
+class AWXDjangoStrategy(DjangoStrategy):
+    """A DjangoStrategy for python-social-auth containing
+       fixes and updates from social-app-django
+    """
+
+    def __init__(self, storage, request=None, tpl=None):
+        super(AWXDjangoStrategy, self).__init__(storage, tpl)
+
+    def request_port(self):
```

Comment:
Nice, lets definitely put a TODO in here to remove this when we bump the social auth code.